app/Helpers/database/game.php

```diff
@@ -102,14 +90,7 @@ function getGameMetadata(
     }
 
     $metricsColumns = '';
-    $metricsJoin = '';
-    $metricsBindings = [];
     if ($metrics) {
-        $parentGame = getParentGameFromGameTitle($gameDataOut['Title'], $gameDataOut['ConsoleID']);
-        $numDistinctPlayersSelector = $parentGame?->players_total ?: getGameData($gameID)['NumDistinctPlayers'];
-        $gameDataOut['ParentGameID'] = $parentGame?->id;
-        $gameDataOut['NumDistinctPlayers'] = $numDistinctPlayersSelector ?? 0;
```

Comment:
`NumDistinctPlayers` is already retrieved via `getGameData` on line 84. The update metrics action should ensure the subset has the same player count as the base set.

`ParentGameID` didn't seem to be used anywhere